src/it/scala/it/pagopa/interop/purposemanagement/projection/cqrs/CqrsProjectionSpec.scala

```diff
@@ -21,8 +21,14 @@ class CqrsProjectionSpec extends ScalaTestWithActorTestKit(ItSpecConfiguration.c
     }
 
     "succeed for event PurposeUpdated" in {
-      val purpose = createPurpose(persistentPurpose)
-      val update = PurposeUpdate(title = "New title", description = "new Description", Some(persistentRiskAnalysisForm))
+      val purpose   = createPurpose(persistentPurpose)
+      val update    = PurposeUpdate(
+        title = "New title",
+        description = "new Description",
+        isFreeOfCharge = false,
+        freeOfChargeReason = None,
```

Comment:
Just to test that the update of new fields is working
```suggestion
        isFreeOfCharge = true,
        freeOfChargeReason = Some("A new reason"),
```